python-backend/app/api/parties/namespace/parties.py

```diff
@@ -1,8 +1,13 @@
 from flask_restplus import Namespace
 
-from ..party.resources.party import PartyResource, ManagerResource
+from ..party.resources.party_resource import PartyResource
+from ..party.resources.manager_resource import ManagerResource
+from ..party_appt.resources.mine_party_appt_resource import MinePartyApptResource
+from ..party_appt.resources.mine_party_appt_type_resource import MinePartyApptTypeResource
 
 api = Namespace('parties', description='Party related operations')
 
 api.add_resource(PartyResource, '', '/<string:party_guid>')
 api.add_resource(ManagerResource, '/managers', '/managers/<string:mgr_appointment_guid>')
+api.add_resource(MinePartyApptResource, '/mines', '/mines')
+api.add_resource(MinePartyApptTypeResource, '/mines/types', '/mines/types/<string:mine_party_appt_type_code>')
```

Comment:
/parties/mines/types - This seems weird to me. Maybe /parties/mines/relationship_types ?